scripts/docker/baseDockerfile

```diff
@@ -40,7 +40,7 @@ WORKDIR /home/idflakies/
 RUN \
   wget http://mirrors.ibiblio.org/apache/maven/maven-3/3.5.4/binaries/apache-maven-3.5.4-bin.tar.gz && \
   tar -xzf apache-maven-3.5.4-bin.tar.gz && mv apache-maven-3.5.4/ apache-maven/ && \
-  echo 'export JAVA_HOME=/usr/lib/jvm/java-8-oracle' >> ~/.bashrc && \
+  echo 'export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/' >> ~/.bashrc && \
```

Comment:
Isn't your change to the JAVA_HOME here specific to how your machine has Java installed in java-8-openjdk-amd64 vs. java-8-oracle?

Reply:
It is specific to the setup in the docker container. Therefore, making the change in the docker file resolves the current inconsistency.